.github/workflows/playwright.yml

```diff
@@ -21,6 +21,7 @@ jobs:
         run: npx playwright test --project replay-chromium --reporter=@replayio/playwright/reporter,line
         env:
           BASE_URL: ${{ github.event.deployment_status.target_url }}
+          REPLAY_PLAYWRIGHT_FIXTURE: 1
```

Comment:
Try adding `DEBUG: replay:*` so we can see what else might be happening in the plugin.

Reply:
Here’s the run: https://github.com/filiphric/things-app-react/actions/runs/6738235217/job/18317390005

Reply:
Yeah, same thing ... I'll have to add some more debugging logs. I can see data flowing from the runner process to the reporter process but then they're being dropped. If you want to dig into it, you're welcome to pull down the `playwright-next` branch of `replay-cli` and build and link locally. It's a bit of work but I can walk you through it if you'd like.

Reply:
I’d love that